docs/JpManualDeploymentGuideScalarDLOnAWS.md

```diff
@@ -0,0 +1,302 @@
+# AWS での Scalar DL 導入方法
+
+Scalar DL は、データベースに依存しない分散型台帳ミドルウェアで、Docker コンテナとして提供されています。  
+様々なプラットフォームに展開可能ですが、高い可用性と拡張性、保守性を実現するために、本番環境ではマネージドサービス上に展開することが推奨されています。  
+本ガイドでは、Scalar DL Ledger を Amazon Web Services (AWS) のマネージドデータベースサービス (DynamoDB) とマネージド Kubernetes サービス (EKS) 上に手動でデプロイする方法を説明します。  
```

Comment:
I feel it's a bit inconsistent because it doesn't explain `Scalar DL Ledger` even though it explains `Scalar DL Auditor`.

Reply:
I see. You are right.

I added the explanation about Scalar DL Ledger and adjusted structure in each document in the following commit.

5c42153620d0733c9b36b66cddd089dd0f3bc26a